config/sigs.json

```diff
@@ -81,6 +81,7 @@
     "0xa8676443",
     "0x301a3720",
     "0x7a1eb1b9",
-    "0xb22f4db8"
+    "0xb22f4db8",
+    "0x3593564c"
```

Comment:
We should also allow the `execute` function without deadline: `0x24856bc3`
https://basescan.org/address/0x198EF79F1F515F02dFE9e3115eD9fC07183f02fC#writeContract